yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -644,11 +650,15 @@ def run(self, info):
         input_files = [filename, *sub_filenames]
 
         opts = [
+            # Attached JSON subtitles don't have a codec id and we have to
+            # instruct FFMPEG to not discard them because of that.
+            '-copy_unknown',
             *self.stream_copy_opts(ext=info['ext']),
```

Comment:
Should `-copy_unknown` be added to `stream_copy_opts` instead? Would that cause any issues?

Reply:
If we are going to implement JSON sub attachments, we would *at least* need to add `-copy_unknown` to *every* ffmpeg PP that would be run after this one. Not sure of the implications of doing so

Reply:
yeah, I realized what you meant shortly after I posted the (now-deleted) comment; momentary lapse in brain function

> I don't think that switching `-ignore_unknown` to `-copy_unknown` would give issues but I want to make you all aware of this.

I wonder what the original reason for going with `-ignore_unknown` over `-copy_unknown` was

Reply:
We dont inspect media file stream contents in tests... I vote for including it, especially if we can do so in a lengthy nightly only period.

Problematic streams should never cause any problems in mkv. One could argue that it might cause problems in other formats. I would test the following case:
mkv with unknown stream => other container

Will the `-copy_unknown` cause problems in that case?

Reply:
Dont rebase/force-push. We will squash, so just add commits on top instead